windows/lib/registerMediaKeys.js

```diff
@@ -0,0 +1,25 @@
+const { globalShortcut } = require('electron');
+const logger = require('./headset-logger');
+
+module.exports = (win) => {
+  globalShortcut.register('MediaPlayPause', () => {
+    logger.media('Executing play-pause media key command');
+    win.webContents.executeJavaScript(`
+        window.electronConnector.emit('play-pause')
+      `);
+  });
+
+  globalShortcut.register('MediaNextTrack', () => {
+    logger.media('Executing play-next media key command');
+    win.webContents.executeJavaScript(`
+        window.electronConnector.emit('play-next')
+      `);
+  });
+
+  globalShortcut.register('MediaPreviousTrack', () => {
+    logger.media('Executing play-previous media key command');
+    win.webContents.executeJavaScript(`
+        window.electronConnector.emit('play-previous')
+      `);
+  });
+};
```

Comment:
It is a duplicate of Mac, but not for Linux and that's the reason why I decided not to move them out to `/shared`. For example, the tray is also common to both Mac and Windows but not Linux, so they're also kept in their own directories.
I feel like unless something is common to all OSs, we shouldn't move them out. Thought?

Reply:
To me, if some piece of code is used by more than one other piece of code. it still considered shared and it's better to avoid duplicated code as much as possible. I think, if you already extract that to a library, it might as well be used by two OSs instead of each to have their own version.

Reply:
Right now, the only thing that's different between Mac and Windows is how we log things. Windows has its own module.
I can a shared library, that not only has the media keys but any other modules that are reused, including the logging. We can just check which OS we're using. The only concern I have is that modules not needed for other OSs will be included anyway, only the logging module has this problem. I'm going to go ahead and do this unless you have other thoughts/concerns.

List of things we can move to a shared module:
- `i18nextConfig.js`
- `headsetTray.js`
- `headsetLogger.js`
- `registerMediaKeys.js`
- All initialization of both windows (the icon is the only difference: ico, icns, png and can be loaded based on OS)
- Loading URL for both windows
- all `ipcMain` events
- Some other event listeners (either windows, apps, etc)

I think this should be a different PR. It'll be easier to move all the code into a single folder, keeping a single Headset app, and loading modules depending on OS. Thoughts?